eclipse-collections/src/main/java/org/eclipse/collections/impl/UnmodifiableRichIterable.java

```diff
@@ -370,6 +405,14 @@ public IntIterable collectInt(IntFunction<? super T> intFunction)
         return this.iterable.collectInt(intFunction, target);
     }
 
+    @Override
+    public <R extends MutableIntCollection> R flatCollectInt(
+            Function<? super T, ? extends IntIterable> function,
+            R target)
```

Comment:
This is a minor formatting inconsistency. All of the other instances have target on the same line as function.